packages/webdriverio/src/utils/refetchElement.js

```diff
@@ -0,0 +1,16 @@
+export default async function refetchElement (currentElement) {
```

Comment:
Why do you use `default` keyword? I mean is it not a [bad practice](https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html)?

Reply:
Why would it be bad practice?

Reply:
I added one link above. And there are several the same articles

Reply:
@christian-bromann You make the call. It seems the majority of the v5 code base uses export default, and this would deviate from it.

Reply:
I am fine having it this way .. this file contains one utility function and only one so it is fine imo to directly export it